test/Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple.Tests/Be.Vlaanderen.Basisregisters.MessageHandling.Kafka.Simple.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Can you change the csproj to use something like this?:

```<Project Sdk="Microsoft.NET.Sdk">
  <Import Project="..\..\packages\Be.Vlaanderen.Basisregisters.Build.Pipeline\Content\Be.Vlaanderen.Basisregisters.Build.Pipeline.Settings.Test.props" />

  <ItemGroup>
    {project reference}
  </ItemGroup>

  <Import Project="..\..\.paket\Paket.Restore.targets" />
</Project>```

with a `paket.references` file